bin/exec.ml

```diff
@@ -93,18 +96,34 @@ let term =
         let prog = Path.extend_basename prog ~suffix:Bin.exe in
         Option.some_if (Path.exists prog) prog
   in
+  (* Good candidates for the "./x.exe" instead of "x.exe" error are executables
+     present at the root of the build dir *)
+  let hints () =
+    let candidates =
+      let path = path_relative_to_build_root "" in
+      let open List in
```

Comment:
Small nitpick: we never open the `List` module. Just for the sake of consistency, let's not do it here either.